Homework_1/Task_3.py

```diff
@@ -0,0 +1,30 @@
+#Напишите код, который запрашивает число и сообщает является ли оно простым или составным.
+#Используйте правило для проверки: “Число является простым, если делится нацело только на единицу и на себя”.
+#Сделайте ограничение на ввод отрицательных чисел и чисел больше 100 тысяч.
+from sympy import *
+def check_input():
+    while True:
+        num = input("Введите число: ")
+        if num.isdecimal() and 0 < (res := int(num)) < 100000:
+            return res
+        print("введите число от 0 до 100 тыс.")
+n = check_input()
```

Comment:
Согласно документации PEP8, функция должна иметь 2 пустых троки до и после своего определения.
```suggestion
from sympy import *


def check_input():
    while True:
        num = input("Введите число: ")
        if num.isdecimal() and 0 < (res := int(num)) < 100000:
            return res
        print("введите число от 0 до 100 тыс.")


n = check_input()
```